src/popup/router/pages/SignTransaction.vue

```diff
@@ -0,0 +1,23 @@
+<script>
+import deeplinkApi from '../../../mixins/deeplinkApi';
+
+export default {
+  mixins: [deeplinkApi],
+  async mounted() {
+    await this.$watchUntilTruly(() => this.$store.state.sdk);
+    const { transaction, networkId, broadcast } = this.$route.query;
+    const signedTransaction = await this.$store.state.sdk
```

Comment:
Proposing to add a draft confirmation here:
```suggestion
    if (networkId !== <current network from store>) {
      await store.dispatch('modals/open', {
        name: 'confirm',
        msg: `This action is being done on ${networkId}, are you sure what to proceed?`,
      });
    }
    const signedTransaction = await this.$store.state.sdk
```